src/Mod/Draft/DraftEdit.py

```diff
@@ -706,78 +712,76 @@ def delPoint(self,event):
-            # do not finish if some trackers are still present
-            if self.trackers == {'object':[]}: self.finish()
-            return
-
-
+            return None
 
-    def update(self,v):
+    def update(self,obj, nodeIndex, v):
         "apply the vector to the modified point and update self.obj"
 
         FreeCAD.ActiveDocument.openTransaction("Edit")
 
-        if Draft.getType(self.obj) in ["Wire","BSpline"]: self.updateWire(v)
-        elif Draft.getType(self.obj) == "BezCurve": self.updateWire(v)
-        elif Draft.getType(self.obj) == "Circle": self.updateCircle(v)
-        elif Draft.getType(self.obj) == "Rectangle": self.updateRectangle(v)
-        elif Draft.getType(self.obj) == "Polygon": self.updatePolygon(v)
-        elif Draft.getType(self.obj) == "Dimension": self.updateDimension(v)
-        elif Draft.getType(self.obj) == "Sketch": self.updateSketch(v)
-        elif Draft.getType(self.obj) == "Wall": self.updateWall(v)
-        elif Draft.getType(self.obj) == "Window": self.updateWindow(v)
-        elif Draft.getType(self.obj) == "Space": self.updateSpace(v)
-        elif Draft.getType(self.obj) == "Structure": self.updateStructure(v)
-        elif Draft.getType(self.obj) == "PanelCut": self.updatePanelCut(v)
-        elif Draft.getType(self.obj) == "PanelSheet": self.updatePanelSheet(v)
-        elif Draft.getType(self.obj) == "Part::Line" and self.obj.TypeId == "Part::Line": self.updatePartLine(v)
-        elif Draft.getType(self.obj) == "Part" and self.obj.TypeId == "Part::Box": self.updatePartBox(v)
+        if Draft.getType(obj) in ["Wire","BSpline"]: self.updateWire(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "BezCurve": self.updateWire(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Circle": self.updateCircle(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Rectangle": self.updateRectangle(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Polygon": self.updatePolygon(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Dimension": self.updateDimension(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Sketch": self.updateSketch(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Wall": self.updateWall(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Window": self.updateWindow(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Space": self.updateSpace(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Structure": self.updateStructure(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "PanelCut": self.updatePanelCut(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "PanelSheet": self.updatePanelSheet(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Part::Line" and self.obj.TypeId == "Part::Line": self.updatePartLine(obj, nodeIndex, v)
+        elif Draft.getType(obj) == "Part" and self.obj.TypeId == "Part::Box": self.updatePartBox(obj, nodeIndex, v)
```

Comment:
Again, even if the old code is in the same line. It's more readable if each branch is in a separate line.